axon/controller/axon_rpyc_controller.py

```diff
@@ -57,6 +58,11 @@ class exposed_ResourceMonitor(ResourceMonitor):
     pass
 
 
+@exposify
+class exposed_Console(Console):
+    pass
+
+
```

Comment:
Let's avoid this for now. as it can lead to major security vulnerability. 
If RPyC is compromised, attackers can hijack session and can explout by command injection.